.mailmap

```diff
@@ -364,6 +364,8 @@ Ashutosh Saboo <ashutosh.saboo96@gmail.com>
 Ashwini Oruganti <ashwini.oruganti@gmail.com>
 Asish Panda <asishrocks95@gmail.com>
 Atharva Khare <khareatharva@gmail.com>
+Au Huishan <151558456+AkierRaee@users.noreply.github.com>
+Au Huishan <huishan_au@outlook.com> pku2100094807 <2100094807@stu.pku.edu.cn>
```

Comment:
There should not be two separate lines where the first entry has different email addresses:
https://github.com/sympy/sympy/blob/0115ea531aa08d1ef6455db8def61e94eea9f561/.mailmap#L129-L148

Reply:
Thank you for pointing out the formatting issue. I've updated it as suggested.